src/utils/changeAttribute/changeEnumAttribute.ts

```diff
@@ -39,7 +43,12 @@ export const changeEnumAttribute = (currentMode?: ModeTypes) => {
 
   body.append(
     'instr',
-    JSON.stringify(['ATOM', 'EDIT', location, filteredAttribute]),
+    JSON.stringify([
+      ActionTypes.ATOM,
+      AtomModifyingActionName.EDIT,
+      location,
+      filteredAttribute,
+    ]),
```

Comment:
Implement a getDsStat method in decision-trees.provider.ts. Get rid of URLParams and fetches

Reply:
decided to fix in another PR